meta-sokol-flex-common/classes/mirrors.bbclass

```diff
@@ -2,11 +2,11 @@
 # SPDX-License-Identifier: MIT
 # ---------------------------------------------------------------------------------------------------------------------
 
-PREMIRRORS:append = " \
+PREMIRRORS:append:sokol-flex = " \
 git://sources.redhat.com/ git://sourceware.org/ \n\
 "
 
-MIRRORS += "\
+MIRRORS:append:sokol-flex = "\
```

Comment:
`:append` without a leading space. Sorry :)

Reply:
Deliberately done; does it take white space or newline here?